apps/portals/e2e/explore.spec.ts

```diff
@@ -185,15 +185,17 @@ const toggleShowHideVisualizationsAndVerify = async (
   const showHideButton = page.getByRole('button', {
```

Comment:
Before this, click "View All Charts" so that the hidden (Age Distribution) chart is now shown. Optionally then click "Hide Charts" to reset to the default view.